src/main/java/org/jabref/gui/maintable/MainTableColumnModel.java

```diff
@@ -35,6 +38,8 @@
         NORMALFIELD("field"),
         SPECIALFIELD("special", Localization.lang("Special"));
 
+        public static List<Type> ICON_COLUMNS = Arrays.asList(EXTRAFILE,FILES,GROUPS,LINKED_IDENTIFIER);
```

Comment:
I'm just using this as a constant (i know, I forgot 'final' here) to spare the subsequent checks on each single column type, if it is an icon column. I implemented this in the first time as an additional argument to the singe Types (boolean isIconColumn), but it would have brought way more complexity. So this way someone later, who wants to add another column type can simply add it in this array (if needed). It does not really make a difference if this is an Array or an EnumSet.